llvm/test/CodeGen/WebAssembly/wasm-eh-prepare.ll

```diff
@@ -245,7 +246,6 @@ bb.true:                                          ; preds = %entry
 bb.true.0:                                        ; preds = %bb.true
   br label %merge
 
-; CHECK:      bb.false
```

Comment:
This was deleted because, in `llc` tests many passes run before `WinEHPrepare` and `WasmEHPrepare`, and some of them did some simplification of the CFG, removing this BB.